src/utils/assets.js

```diff
@@ -422,7 +422,8 @@ export const sortInvestmentPositions = (positionsInfo: AccountInvestmentPosition
   return newArr;
 };
 
-export const isSameAsset = (a: Asset, b: Asset) => a.address?.toLowerCase() === b.address?.toLowerCase();
+export const isSameAsset = (a: Asset, b: Asset) =>
+  a.symbol === b.symbol && a.address.toLowerCase() === b.address.toLowerCase();
```

Comment:
We should just be checking the address when comparing two assets - the symbol _could_ potentially be different depending where the asset is pulled from. The address is always unique

Also any reason for removing the null check `?`?
```suggestion
 a?.address?.toLowerCase() === b?.address?.toLowerCase();
```